rviz_common/include/rviz_common/message_filter_display.hpp

```diff
@@ -140,6 +141,11 @@ class MessageFilterDisplay : public _RosTopicDisplay
 
   void updateMessageQueueSize()
   {
+    if (topic_property_->isEmpty()) {
```

Comment:
I think there's another edge case where an invalid topic name is provided.

I didn't check, but I guess the real reason for the segfault is accessing a nullptr (`tf_filter_`). Rather than trying to handle the same cases related to topic name in `subscribe()`, can we just check if `tf_filter_` if valid?

e.g.

```c++
if (tf_filter_) {
  tf_filter_->setQueueSize(static_cast<uint32_t>(message_queue_property_->getInt()));
}
```

Reply:
That works! Thanks for the better solution!